meringue-maven-plugin/src/main/java/edu/neu/ccs/prl/meringue/CampaignReport.java

```diff
@@ -55,9 +55,8 @@ public void write(File coverageFile, File failuresFile) throws IOException {
         }
     }
 
-    public void writeHtmlReport(String testDescription, File reportDir) throws IOException {
-        calculator.createHtmlReport(lastExecData == null ? new byte[0] : lastExecData, testDescription, sources,
-                                    reportDir);
+    public void writeReport(String testDescription, File reportDir, ReportFormat f) throws IOException {
+        calculator.createReport(lastExecData == null ? new byte[0] : lastExecData, testDescription, sources, f.createVisitor(reportDir));
```

Comment:
Line may be too long; does Meringue use Checkstyle or something similar?

Reply:
Yes. This code passes checkstyle.